### PR TITLE
1892 tracing

### PR DESCRIPTION
**Pull request by tcharding**
Convert code base to use tokio-rs tracing library.  With this applied (after running btc_eth/happy e2e test):
```
Jan 29 14:12:50.122  INFO cnd::trace: Initialized tracing with level: TRACE
Jan 29 14:12:50.122  INFO cnd::seed: RootSeed file parent directory does not exist, creating recursively: /tmp/tmp-125556e02qp5UNTGu/seed.pem
Jan 29 14:12:50.122  INFO cnd::seed: No seed file found, creating at: /tmp/tmp-125556e02qp5UNTGu/seed.pem
Jan 29 14:12:50.288  INFO cnd::db: SQLite database file: /tmp/tmp-125556e02qp5UNTGu/cnd.sqlite
Jan 29 14:12:50.290  INFO cnd::network: Starting with peer_id: QmafYkp7uHUj3FghWAbPZDbRTEFtJRdH3nQF5kXBYdXQYt
Jan 29 14:12:50.293 DEBUG cnd::load_swaps: loading swaps from database ...
Jan 29 14:12:50.295  INFO cnd: Starting HTTP server on 0.0.0.0:42789
Jan 29 14:12:53.583 TRACE cnd::http_api::routes::rfc003::handlers::post_swap: initiating new request: 932784b5-8ed9-47c1-8219-f216a9e0a017
Jan 29 14:12:53.589  INFO http: 127.0.0.1:39774 "POST /swaps/rfc003 HTTP/1.1" 201 "-" "axios/0.19.0" 39.353322ms    
Jan 29 14:12:53.589 DEBUG cnd::network: Making swap request to QmQskR8UngQvXfHrvrubAACkhQNVo4D2dPowjW81hWUxSu@/ip4/127.0.0.1/tcp/33085: SwapId(932784b5-8ed9-47c1-8219-f216a9e0a017)
```

And we get the same total number of log lines from running btc_eth/happy e2e test as before.

Fixes: #1892

### CODE REVIEW

File: cnd/src/trace.rs [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+    subscriber::set_global_default(subscriber)?;
+    info!("Initialized tracing with level: {}", level);
+
+    Ok(())
+}
+
+fn level_from_level_filter(level: LevelFilter) -> Level {
+    match level {
+        LevelFilter::Off => Level::ERROR,
```
Comment (thomaseizinger) [inline cnd/src/trace.rs]: Why are we not respecting `off`? Could just not initialize anything?

File: cnd/src/main.rs [hunk 2]
```diff
@@ -48,8 +50,10 @@ fn main() -> anyhow::Result<()> {
         process::exit(0);
     }
 
-    let base_log_level = settings.logging.level;
-    logging::initialize(base_log_level, settings.logging.structured)?;
+    // We still want upstream library log messages, just only at Info level.
+    LogTracer::init_with_filter(LevelFilter::Info)?;
```
Comment (thomaseizinger) [inline cnd/src/main.rs]: I would see this as part of the initialization, no?
Why add this level of detail to `main`?

File: cnd/src/trace.rs (hunk 1 above)
Comment (tcharding) [reply cnd/src/trace.rs]: Good idea, will implement.

File: cnd/src/main.rs (hunk 2 above)
Comment (tcharding) [reply cnd/src/main.rs]: Good point, thanks.  Implemented.

**Pull request merged**